certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk3226/FK3226CertificateSummaryProvider.java

```diff
@@ -10,7 +15,36 @@ public class FK3226CertificateSummaryProvider implements CertificateSummaryProvi
   public CertificateSummary summaryOf(Certificate certificate) {
     return CertificateSummary.builder()
         .label("Avser diagnos")
-        .value("Namn på exempeldiagnos")
+        .value(getValue(certificate))
         .build();
   }
+
+  private String getValue(Certificate certificate) {
```

Comment:
Shouldn't this logic be placed in a general place so it can be reused by other certificates who have diagnosis as summary

Reply:
Agreed, but does it make sense to do that after we introduce the other certificate with diagnosis?

Reply:
Add common package with common diagnosis logic.